docs/VictoriaLogs/README.md

```diff
@@ -117,7 +117,7 @@ Set the `-retentionPeriod` to some big value (e.g. `100y` - 100 years) if logs s
 For example:
 
 ```sh
-/path/to/victoria-logs -retention.maxDiskSpaceUsageBytes=10TiB -retention=100y
+/path/to/victoria-logs -retention.maxDiskSpaceUsageBytes=10TiB -retentionPeriod=100y
```

Comment:
this change is unrelated. Could you please remove it?

Reply:
removed.